usr/sbin/p11sak/p11sak.mk

```diff
@@ -9,4 +9,4 @@ usr_sbin_p11sak_p11sak_CFLAGS  =			\
 	-I${srcdir}/usr/sbin/p11sak
 
 usr_sbin_p11sak_p11sak_SOURCES =			\
-	usr/sbin/p11sak_tool/p11sak.c
+	usr/sbin/p11sak/p11sak.c
```

Comment:
Squash this change with your very first commit, too

Reply:
done